src/common/Brush/Brush.tsx

```diff
@@ -211,7 +211,9 @@ export class Brush extends PureComponent<BrushProps, BrushState> {
           }}
         >
           {children}
-          {!disabled && (
+          {disabled ? (
+            <rect ref={(ref) => (this.ref = ref)} />
+          ) : (
```

Comment:
@amcdnl i tried adding this logic within the existing <rect> element, but there's some rendering bit between React 17/18 incompatibility that's causing issues. so this was the cleaner fix for now